ext/ftp/tests/ftp_append.phpt

```diff
@@ -13,15 +13,19 @@ if (!$ftp) die("Couldn't connect to the server");
 
 var_dump(ftp_login($ftp, 'user', 'pass'));
 
-@unlink(__DIR__.'/ftp_append_foobar');
+$fooPath = __DIR__ . '/ftp_append_foo';
+file_put_contents($fooPath, 'foo');
+var_dump(ftp_append($ftp, 'ftp_append_foobar', $fooPath, FTP_BINARY));
+unlink($fooPath);
```

Comment:
Should those unlink calls be moved into --CLEAN-- ?

Reply:
I put it close to where the file is created so I could refer to the same path easily with a variable. But I can move it to a --CLEAN-- if you think that's better.

Reply:
`CLEAN` makes it clearer at a glance I think ...